apps/contract-sync-job/src/main.ts

```diff
@@ -11,6 +11,7 @@ import { GraphQLClient } from 'graphql-request';
 
 const protcolUrl = process.env.PROTOCOL_URL;
 const SUBGRAPH_ENDPOINT = process.env.SUBGRAPH_URL;
+const KEVIN_MALONE_TOKEN = process.env.KEVIN_MALONE_TOKEN;
```

Comment:
We probably shouldn't be reusing tokens across different jobs. We should create a new token in protocol api called CONTRACT_SYNC_TOKEN and use that in this job